test/core/fixtures/transformation/es7.function-bind/call/expected.js

```diff
@@ -0,0 +1,8 @@
+"use strict";
+
+var _context;
+
+ns.obj.func.call(ctx);
+(_context = ns.obj).func.call(_context);
+
+ns.obj1.func.call(ns.obj2);
```

Comment:
It just occurred to me that in the following case:

```js
ns.obj2.prop::ns.obj1.func()
```

If we translate to

```js
ns.obj1.func.call(ns.obj2.prop);
```

we could have some order-of-operations issues where `obj2` needs to be looked up before `obj1`.  It might need to translate to something like:

```js
(_context = ns.obj2, ns).obj1.func.call(_context.prop);
```

esdown gets this wrong at the moment.